src/Workspaces/CSharp/Portable/Formatting/Rules/IndentBlockFormattingRule.cs

```diff
@@ -150,6 +152,14 @@ private void AddAlignmentBlockOperation(List<IndentBlockOperation> list, SyntaxN
                 case ImplicitArrayCreationExpressionSyntax implicitArrayCreation when implicitArrayCreation.Initializer != null:
                     SetAlignmentBlockOperation(list, implicitArrayCreation.NewKeyword, implicitArrayCreation.Initializer.OpenBraceToken, implicitArrayCreation.Initializer.CloseBraceToken, IndentBlockOption.RelativeToFirstTokenOnBaseTokenLine);
                     return;
+                case CSharpSyntaxNode syntaxNode when syntaxNode.IsKind(SyntaxKindEx.SwitchExpression):
+                    SetAlignmentBlockOperation(
+                        list,
+                        syntaxNode.GetFirstToken(),
+                        syntaxNode.ChildNodesAndTokens().First(child => child.IsKind(SyntaxKind.OpenBraceToken)).AsToken(),
```

Comment:
can we just cast node to switchexpression and use open brace and close brance token directly ?

Reply:
No, `SwitchExpressionSyntax` does not exist in the Code Style layer.